### PR TITLE
Added test coverage command

### LINKED ISSUE #127: Add automated test coverage to presubmit
Use https://github.com/romeovs/lcov-reporter-action

### PR DESCRIPTION
**Pull request by kebrahim**
Works on #127 

Will add the presubmit in a separate PR

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "lint:js": "eslint --ignore-path ../.gitignore --ext .js,.jsx .",
     "lint:js:fix": "eslint --ignore-path ../.gitignore --ext .js,.jsx . --fix",
     "start": "react-scripts start",
-    "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
+    "test": "react-scripts test --coverage --env=jest-environment-jsdom-sixteen",
```
Comment (marcja) [inline client/package.json:26]: This is a bad idea. Create a separate "test:cov" target. Coverage radically slows test execution, which isn't great for test-driven development.

File: .github/workflows/client.yml [hunk 2]
```diff
@@ -5,34 +5,35 @@ name: Validate `client`
...
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install yarn
+        run: npm install -g yarn
+      - run: yarn install
+      - run: yarn lint
+      - run: yarn run build
```
Comment (marcja) [inline .github/workflows/client.yml:34]: `yarn build`

Comment (kebrahim) [reply .github/workflows/client.yml]: reverted

File: client/package.json (hunk 1 above)
Comment (kebrahim) [reply client/package.json]: done

**Pull request merged**